### ISSUE TITLE
Dev install fails on R 3.6.0

### ISSUE THREAD

**Issue by WibbletheDuck** [MacoskoLab/liger#63]
In earlier versions of R, `devtools::install_github("liger")` throws warnings,
```
Warning: /tmp/RtmpQQenn7/Rbuild3338f90697/liger/man/liger-package.Rd:26: All text must be in a section
Warning: /tmp/RtmpQQenn7/Rbuild3338f90697/liger/man/liger-package.Rd:27: All text must be in a section
```
but the install succeeds.  Now in 3.6.0 the warning gets upgraded to an error,
```
Error : (converted from warning) /tmp/RtmpUvcD5m/Rbuild325d7f4ef7/liger/man/liger-package.Rd:26: All text must be in a section
ERROR: installing Rd objects failed for package ‘liger’
```
and thus the install fails.  I can workaround by setting `R_REMOTES_NO_ERRORS_FROM_WARNINGS` to true, but it would be nice to fix the issue at the source.

**Comment by vkozareva** [comment]
Thanks for notifying us of this -- this will be fixed in the next minor release, coming out in the next few days!

**Comment by desmondramirez** [comment]
Hi folks,

I am having the same issue using trying to install Liger.

`Error : (converted from warning) /private/var/folders/0c/2hw_wqys6gs7s02860y8tzq00000gn/T/Rtmp7pKfqi/Rbuild8d4186f782b/liger/man/liger-package.Rd:26: All text must be in a section
ERROR: installing Rd objects failed for package ‘liger’`

Is there any new on when the next minor release might be coming out? Otherwise I will try using an earlier version of R to install-- I'm eager to try Liger out!

**Comment by karthikshekhar** [comment]
Hello Velina,

Any thoughts on when the fix for R 3.6.0 would be released? Thanks!

Karthik

**Comment by vkozareva** [comment]
Went ahead with this small fix -- full minor release is still coming out later today!
